api/__init__.py

```diff
@@ -0,0 +1,40 @@
+import requests
+
+DEFAULT_CONTENT_TYPE = "application/json"
+DEFAULT_ACCEPT = "application/vnd.ceph.api.v1.0+json"
+DEFAULT_VERIFY = True
+
+
+class Api:
+    def __init__(self, url, api):
+        requests.packages.urllib3.disable_warnings(
+            requests.packages.urllib3.exceptions.InsecureRequestWarning
+        )
+
+        self.url = f"{url}/${api}"
+
+    def get(self):
+        pass
+
+    def post(
+        self,
+        data,
+        verify=DEFAULT_VERIFY,
+        content_type=DEFAULT_CONTENT_TYPE,
+        accept=DEFAULT_ACCEPT,
+    ):
+        headers = {
+            "Content-Type": content_type,
+            "Accept": accept,
+        }
+
+        return requests.post(url=self.url, verify=verify, headers=headers, data=data)
+
+    def put(self):
+        pass
```

Comment:
please define everything

Reply:
Added these values as placeholders for now. Will define them once add methods to automate.